### PR TITLE
Feature/jcs location comp

### LINKED ISSUE #39: Combine Data fetch to one data file
- [ ] Combine data fetch to a singular object to remove any data we dont need.
- [ ] What functionality do we need to manipulate the data for display?
- [ ] What data are we getting?
- [ ] Any data we may be missing?

### PR DESCRIPTION
**Pull request by JoshSevy**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Data clean up function to bring all needed data to one data point
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other, Please explain:
`Your message here`
## Has This Been Tested?
- [ ] No
- [x] Yes
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:
Functionality of the call function was done with a console log in the browser- I left the console log in since it is such a large data file. I plan to remove it once all the data is being rendered
# Message for reviewer:
Sorry it didn't go as planned, i thought it would of cleaned up the data a bit more, in retrospect I should of just done this first and went back and refactored. Sorry for the delay.
# Checklist:
- [ ] My code has no unused/commented out code 
--- All of location is commented out so that the app still loads 
--- until I get the data passed in and fix the typescript errors it will crash the app
- [ ] My code has no console logs
--- console log in getAllData so that the data coming back can be referenced
- [x] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/App/App.tsx [hunk 1]
```diff
@@ -9,15 +9,20 @@ import './App.scss'
 
 // import { getAirQualityData } from '../helpers/apiCalls'
 import { getCoordinates } from '../helpers/apiCalls'
+import { getAllData } from "../helpers/dataCleaner"
```
Comment (leighlars) [inline src/App/App.tsx:12]: yesssss

File: src/Location/Location.tsx [hunk 2]
```diff
@@ -11,84 +11,108 @@ import wind from '../assets/wind.png'
...
+		windDirection?: string,
+		precipitation?: number,
+		allergyLevel?: number,
+		discussion?: string,
+  }
 
-		</section>
-	)
-}
+// const Location = ({locationData, currentWeather, weeklyWeather, currentAir}) => {
```
Comment (JoshSevy) [inline src/Location/Location.tsx:34]: Had to comment out location data it was causing some errors I didnt want to hold you all up with

File: src/Location/Location.tsx [hunk 3]
```diff
@@ -11,84 +11,108 @@ import wind from '../assets/wind.png'
...
+// 	const [allData, setAllData] = useState()
 
-export default Location
\ No newline at end of file
+// 	// setAllData(getAllData('denver'))
+	
+// 	return (
```
Comment (leighlars) [inline src/Location/Location.tsx:39]: I am so impressed omg

File: src/helpers/dataCleaner.tsx [hunk 4]
```diff
@@ -0,0 +1,71 @@
+import { getCoordinates, getWeatherData, getAirQualityData} from './apiCalls';
```
Comment (leighlars) [inline src/helpers/dataCleaner.tsx:1]: this is incredible. thank you!

**Comment by leighlars** [comment]
myyyy goodness this is impressive. thank you!

File: src/helpers/dataCleaner.tsx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+
+export const getAllData = async (query: string) => {
```
Comment (ErinUntermeyer) [inline src/helpers/dataCleaner.tsx:4]: This looks SO CLEAN! Thank you so much for sorting through all the data we get back and giving it to us a neat little bow to work with! Great job.

File: src/helpers/dataCleaner.tsx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+  const data: any = {};
+
+  // const kelvinToFahrenheit = (kelvin: number) => {
+  //   return (kelvin - 273.15) * 1.8 + 32
+  // }
+
+  // const kelvinToCelcius = (kelvin: number) => {
```
Comment (JoshSevy) [inline src/helpers/dataCleaner.tsx:11]: These functions are still gonna be used to do the conversions on the display items.

File: src/Location/Location.tsx [hunk 7]
```diff
@@ -11,84 +11,108 @@ import wind from '../assets/wind.png'
...
-					</tbody>
-				</table>
-			</div>
-			<div className='info-box hourly-forecast'>
-				<h4 className='info-box-header'>HOURLY FORECAST</h4>
-			</div>
-			<div className='info-box weekly-forecast'>
-				<h4 className='info-box-header'>WEEKLY FORECAST</h4>
-			</div>
+interface LocationProps {
```
Comment (ErinUntermeyer) [inline src/Location/Location.tsx:16]: Look at those PROPS! Looks great.

**Comment by ErinUntermeyer** [comment]
Great job Josh! I'm excited to get some real data displaying now!

**Pull request merged**